fixtures/codegen/todolist.simplepath.yml

```diff
@@ -33,6 +33,7 @@ paths:
           in: path
           type: integer
           description: an integer  property
+          required: true
```

Comment:
required: true is implied by in: path   it doesn't have to be provided per spec

Reply:
Perhaps it should, but it is not working like this currently.
You've just opened a new issue in the go-openapi/spec package... 💯 

required: true is not implied and (currently) must be provided explicitly. This check is carried on by validate.

Reply:
No, it is not implied by path.  It is required and must be true.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md

>Determines whether this parameter is mandatory. If the parameter is in "path", this property is **required** and its value MUST be true. Otherwise, the property MAY be included and its default value is false.